code_schemes/rqas/aik/aik_incidents_of_violence_and_polarisation_2.json

```diff
@@ -13,6 +13,94 @@
             "VisibleInCoda": true,
             "Shortcut": "e"
         },
+        {
+            "CodeID": "code-a8e36bc1",
+            "CodeType": "Normal",
+            "DisplayText": "yes, no reason",
+            "NumericValue": 1,
+            "StringValue": "yes_no_reason",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-9eedab76",
+            "CodeType": "Normal",
+            "DisplayText": "yes, there is violence",
+            "NumericValue": 2,
+            "StringValue": "yes_there_is_violence",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-d23443f9",
+            "CodeType": "Normal",
+            "DisplayText": "yes, incitement from politicians",
+            "NumericValue": 3,
+            "StringValue": "yes_incitement_from_politicians",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-19635ab9",
+            "CodeType": "Normal",
+            "DisplayText": "yes, tribalism and favouritism",
+            "NumericValue": 4,
+            "StringValue": "yes_tribalism_and_favouristism",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-f26b26a9",
+            "CodeType": "Normal",
+            "DisplayText": "no, no reason",
+            "NumericValue": 5,
+            "StringValue": "no_no_reason",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-0862f780",
+            "CodeType": "Normal",
+            "DisplayText": "no, no violence",
+            "NumericValue": 6,
+            "StringValue": "no_no_violence",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-353f73ab",
+            "CodeType": "Normal",
+            "DisplayText": "no signs of violence yet",
+            "NumericValue": 7,
+            "StringValue": "no_signs_of_violence_yet",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-87c29382",
```

Comment:
Does it help when labelling to sort these by yes codes first then no codes? If not, I'll stop suggesting it :)

Reply:
Yeah, it helps. Updated thanks.